library/libraries.js

```diff
@@ -1945,5 +1945,19 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
             }
             return false;
         }
+    },
+    'NitroPack':{
+        id:'nitropack',
+        icon:'nitropack',
+        url: 'https://nitropack.io/',
+        npm: null,
+        test: async function (win) {
+            const nitroPackGenerator = !!document.querySelector('meta[name="generator"][content="NitroPack"]');
+
+            if ( nitroPackGenerator ) {
+                return { version: UNKNOWN_VERSION };
```

Comment:
is the version available in your generator tag? if not, I suggest you consider adding it for reporting purposes then adding extraction here (see "Drupal" above for an example).

Reply:
It is a rolling release cloud system with no single version number :)